From 6f60bb734892e4a6cb9ba0c48660814160ee49d2 Mon Sep 17 00:00:00 2001
From: Wenchao Hao <haowenchao@huawei.com>
Date: Mon, 6 Dec 2021 10:20:02 +0800
Subject: [PATCH] Remove iscsid.service's dependence of iscsi-init.service

iscsi-init.service is aimed to generate initiatorname.iscsi,
this name would be generated by spec file in openeulr, so we
excluded the iscsi-init.service.

Signed-off-by: Wenchao Hao <haowenchao@huawei.com>
---
 etc/systemd/iscsid.service | 1 -
 1 file changed, 1 deletion(-)

diff --git a/etc/systemd/iscsid.service b/etc/systemd/iscsid.service
index 3fd7dd3..70c7fbd 100644
--- a/etc/systemd/iscsid.service
+++ b/etc/systemd/iscsid.service
@@ -5,7 +5,6 @@ DefaultDependencies=no
 After=network-online.target iscsiuio.service iscsi-init.service
 Before=remote-fs-pre.target
 Wants=remote-fs-pre.target
-Requires=iscsi-init.service
 
 [Service]
 Type=notify
-- 
1.8.3.1

